packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -125,6 +124,10 @@ const updateSession = (
 
     const sessionDeadline = sessionDeadlineRaw + timeLeftTillRoundEnd;
 
+    if (typeof roundPhase !== 'undefined' && roundPhase !== phase) {
+        account.session.sessionPhaseQueue = [];
```

Comment:
when a new phase starts you are cleaning the queue but at the same time you are setting the queue on line 288. Perhaps there should be just 1 trigger for modifying the queue on round phase change?

Reply:
e497b9aab9f2af81b43ea1011049d3cc16223f13